docs/public/consortium_map.jsx

```diff
@@ -1 +1,19 @@
-<ConsortiumMap />
\ No newline at end of file
+<>
+    <p>The SMaHT (Somatic Mosaicism across Human Tissues) Network is a new NIH initiative to identify non-inherited, somatic variations in human genomes over time and to understand the role of such genetic variation in human health, development, aging, and disease.</p>
+
+    <p>The Network consists of:</p>
+    <ol>
+        <li>A Tissue Procurement Center that procures tissue samples from post-mortem donors</li>
+        <li>Five Genome Characterization Centers that generate sequencing data</li>
+        <li>A Data Analysis Center that uniformly processes the SMaHT data</li>
+        <li>14 Tools and Technology Development Projects what focus on developing novel experimental assays and computational methods to detect somatic mutations.</li>
```

Comment:
Thinking this is supposed to read: "14 Tools and Technology Development Projects **that** focus..."

Reply:
Thank you for catching this!